### ISSUE TITLE
v2.0.4 binaries do not have version number

### ISSUE THREAD

**Issue by pocke** [alecthomas/gometalinter#439]
The released v2.0.4 binaries print `master` as version.

```bash
# Download the binary from here https://github.com/alecthomas/gometalinter/releases/tag/v2.0.4
$ wget https://github.com/alecthomas/gometalinter/releases/download/v2.0.4/gometalinter-2.0.4-linux-amd64.tar.gz
$ tar xvf gometalinter-2.0.4-linux-amd64.tar.gz
gometalinter-2.0.4-linux-amd64/COPYING
gometalinter-2.0.4-linux-amd64/README.md
gometalinter-2.0.4-linux-amd64/gometalinter
gometalinter-2.0.4-linux-amd64/gocyclo
gometalinter-2.0.4-linux-amd64/nakedret
gometalinter-2.0.4-linux-amd64/misspell
gometalinter-2.0.4-linux-amd64/govet
gometalinter-2.0.4-linux-amd64/gas
gometalinter-2.0.4-linux-amd64/golint
gometalinter-2.0.4-linux-amd64/ineffassign
gometalinter-2.0.4-linux-amd64/goconst
gometalinter-2.0.4-linux-amd64/errcheck
gometalinter-2.0.4-linux-amd64/maligned
gometalinter-2.0.4-linux-amd64/unconvert
gometalinter-2.0.4-linux-amd64/dupl
gometalinter-2.0.4-linux-amd64/structcheck
gometalinter-2.0.4-linux-amd64/varcheck
gometalinter-2.0.4-linux-amd64/safesql
gometalinter-2.0.4-linux-amd64/deadcode
gometalinter-2.0.4-linux-amd64/lll
gometalinter-2.0.4-linux-amd64/goimports
gometalinter-2.0.4-linux-amd64/gotype
gometalinter-2.0.4-linux-amd64/gosimple
gometalinter-2.0.4-linux-amd64/megacheck
gometalinter-2.0.4-linux-amd64/staticcheck
gometalinter-2.0.4-linux-amd64/unused
gometalinter-2.0.4-linux-amd64/interfacer
gometalinter-2.0.4-linux-amd64/unparam
$ ./gometalinter-2.0.4-linux-amd64/gometalinter --version
master
```

I don't check binaries for other platform, but I guess they also print `master`.

Can you rebuild the v2.0.4 binaries?

Note
===

v2.0.3 binary print correct version number.

```bash
$ ./gometalinter-v2.0.3-linux-amd64/gometalinter --version
v2.0.3
```

**Comment by alecthomas** [comment]
Changed from custom script to goreleaser. Fixed in v2.0.5.

**Comment by pocke** [comment]
Thank you!
